src/openms/source/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.cpp

```diff
@@ -161,71 +161,24 @@ namespace OpenMS
   void MRMTransitionGroupPicker::calculateBgEstimationExact_(const MSChromatogram& chromatogram,
```

Comment:
This method should go away with PeakIntegrator.

Reply:
Shouldn't every method stay, and internally calling PeakIntegrator's methods?

Reply:
No.  The other methods should be replaced similar to how you are calling the method to calculate peakShapMetrics.